tests/README.md

```diff
@@ -201,7 +206,7 @@ For executing this test, first make sure you are in the correct environment as d
 pytest tests/unit/test_notebooks_python.py::test_sar_single_node_runs
 ```
 
-### Developing smoke and integration tests with Papermill
```

Comment:
We need to update this link: `The complete code can be found in [tests/smoke/test_notebooks_python.py](smoke/test_notebooks_python.py):`

Reply:
Done